src/components/popover/popover.tsx

```diff
@@ -202,7 +202,7 @@ export class Popover implements FloatingUIComponent, OpenCloseComponent {
   }
 
   /**
-   * When true, disables automatically toggling the component when its `referenceElement` has been triggered. This property can be set to "true" to manage when a popover is open.
+   * When `true`, disables automatically toggling the component when its `referenceElement` has been triggered. This property can be set to `true` to manage when a popover is open.
```

Comment:
Should "a popover" be "the component" here? Also `open` is a prop should that be in backticks?

Reply:
Updated popover to the component, good catch! 👍🏻 

Since open doesn't relate directly to the property in this case, and referencing it as being accessible to be open/closed via the `triggerDisabled` property - leaving it as open (without backticks).